### PR TITLE
Add AllowedHostsOriginValidator to examples.

### PR DESCRIPTION
**Pull request by collinanderson**
https://github.com/django/channels/issues/1657

### CODE REVIEW

File: docs/topics/sessions.rst [hunk 1]
```diff
@@ -24,16 +24,19 @@ in your ``asgi.py``:
...
     application = ProtocolTypeRouter({
 
-        "websocket": SessionMiddlewareStack(
-            URLRouter([
-                url(r"^front(end)/$", consumers.AsyncChatConsumer.as_asgi()),
-            ])
+        "websocket": AllowedHostsOriginValidator(
+            SessionMiddlewareStack(
+                URLRouter([
+                    url(r"^front(end)/$", consumers.AsyncChatConsumer.as_asgi()),
```
Comment (smithdc1) [inline docs/topics/sessions.rst:37]: `url` is deprecated/removed from Django?

Maybe worth updating to use `path()`?

Comment (collinanderson) [reply docs/topics/sessions.rst:37]: Ok here's a separate PR for removing url(): #1826

**Review by carltongibson — APPROVED** [review]
OK, yes, let's add it... — happy to merge if you rebase. Thanks @collinanderson! 👍

**Comment by collinanderson** [comment]
Ok I rebased.

**Review by carltongibson — APPROVED** [review]
Super. Thanks @collinanderson! 🎁

**Pull request merged**